src/resources/views/crud/datatable/datatable.blade.php

```diff
@@ -0,0 +1,148 @@
+@php
+    // Define the table ID - use the provided tableId or default to 'crudTable'
+    $tableId = $tableId ?? 'crudTable';
+@endphp
+
```

Comment:
The table count doesn't show when the component is used alone - it only shows in the main ListOperation view.

We need to move the table title and subtitle to the component.